doc/sphinx/language/extensions/evars.rst

```diff
@@ -16,10 +16,10 @@ Existential variables
 |Coq| terms can include existential variables that represent unknown
 subterms that are eventually replaced with actual subterms.
 
-Existential variables are generated in place of unsolvable implicit
+Existential variables are generated in place of unsolved implicit
 arguments or “_” placeholders when using commands such as ``Check`` (see
 Section :ref:`requests-to-the-environment`) or when using tactics such as
-:tacn:`refine`, as well as in place of unsolvable instances when using
+:tacn:`refine`, as well as in place of unsolved instances when using
 tactics such that :tacn:`eapply`. An existential
 variable is defined in a context, which is the context of variables of
 the placeholder which generated the existential variable, and a type,
```

Comment:
Looks good.

It's hard to read this section without finding more text to improve.  The text from "An existential variable is defined..." through the next paragraph is really poorly worded.  It probably deserves a very short example (maybe inline in the text).
If you want to fix that too, great and if not, I'll merge this as it is.

Reply:
“It's hard to read this section without finding more text to improve.” → I fully agree! I started on this because I noticed a typo… and then I started to understand that I should probably also rephrase a sentence that was really difficult to read, and it keeps going 😅​

To be honest, I don’t think that I’m the most relevant person to improve these sentences: if you don’t mind me to pause on this PR for now, I would prefer to. Thanks for the comments! ☺

Reply:
I will merge shortly then.

> I don’t think that I’m the most relevant person to improve these sentence

If you're interested in improving wording or the doc in general, don't let that stop you.  It works best with a knowledgeable reviewer.  I've been slowly improving the doc over the last year even though there are many parts of Coq, some basic, that I didn't understand at the outset.

In many places it's hard to figure out what the text means.  Updating wording so it's clear is progress, even if the clear statement is not 100% correct, because the added clarity makes it possible for readers to identify the errors.